cmd/pipeline/deploy.go

```diff
@@ -38,13 +38,17 @@ import (
 	"github.com/okteto/okteto/pkg/repository"
 	"github.com/okteto/okteto/pkg/types"
 	"github.com/spf13/cobra"
-	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/api/core/v1"
 )
 
 const (
 	dependencyEnvTemplate = "OKTETO_DEPENDENCY_%s_VARIABLE_%s"
 )
 
+var (
+	errUnableToReuseParams = errors.New("development environment not found: unable to use --reuse-params option")
```

Comment:
i would need some 👁️ for this copy. Intent is to inform the user that if using the flag but no configmap is found, then they should not use the flag and make a regular okteto pipeline deploy

cc @omnomagonz for some copy 🪄 magic

Reply:
```suggestion
	errUnableToReuseParams = errors.New("configmap not found: unable to use --reuse-params option")
```

Maybe I'm not understanding this context but would this suggestion make sense?

I'm assuming the original copy said "development environment" since that's the user-facing resource that's "not found", but would it be more clear to say what we mean about the missing configmap?

"More clear" here could mean easier to tell us "Hey I'm getting this error about a missing configmap when trying to pipeline deploy"

Please correct me if I'm missing the mark 🙏

Reply:
thanks Matt! The configmap is the k8s component we use to "model" a dev environment. This stores the data that "defines" a dev environment in terms of repository, branch, etc... so... technical interface is a configmap but "popular" name would be dev environment, also known as gitdeploy....

Reply:
@teresaromero Thanks so much for the clarification 🙇 

If that's the case then I'm not sure I'd suggest a change here. It seems your original copy conveys what's happening at least in this context.

Eventually we probably want a specific error code but I know that's a separate, documented initiative that lives elsewhere 😄 

I'd say we ship this as-is unless someone else objects 👍